src/api/chat-post-with-weather-info.js

```diff
@@ -0,0 +1,84 @@
+const https = require("https");
+const TOKEN = process.env.LINE_ACCESS_TOKEN;
+
+const chatPostWithWeatherInfo = function(event) {
+     // Text and type information
+    const replyToken = event.replyToken;
+    const messageType = event.message.type;
+    const messageText = event.message.text;
+
+    if (messageType !== "text") {
+        return;
+    }
+    
+    // URL is sent to Open weather API
+    const APIKEY = process.env.OPEN_WEATHER_API_KEY;
+    const unit = "metric";
+    const url = "https://api.openweathermap.org/data/2.5/forecast?q=" + messageText + "&appid=" + APIKEY + "&units=" + unit;
+
+    // Set information
+    https.get(url, function(response) {
+        response.on("data", function(data) {
+            const weatherData = JSON.parse(data);
+            console.log(weatherData);
+
+            // Set four-day weather forecast information to message array
+            const message = [];
+            for (let i = 0; i < 32; i += 8) {
+                const forecastDate = weatherData.list[i].dt_txt;
+                const dataRemovedTime = forecastDate.substring(0, forecastDate.length - 9);
+                const weatherDescription = weatherData.list[i].weather[0].main;
+                const forecastObj = {
+                                    "type": "text",
+                                    "text": `${dataRemovedTime}: ${weatherDescription}`
+                                    };
```

Comment:
Are you using prettier? Can you setup if you are not?

Reply:
My prettier was not work well but now it's work well after fixing setting.